docs/database-collection/time-series.txt

```diff
@@ -0,0 +1,186 @@
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: chronological, data format, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+odm-short+} to create
+and interact with **time series collections**. These collections store
+time series data, which is composed of the following components:
+
+- Measured quantity
+- Timestamp for the measurement
+- Metadata that describes the measurement
+
+The following table describes sample situations for which you can store time
+series data:
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Situation
+     - Measured Quantity
+     - Metadata
+
+   * - Recording monthly sales by industry
+     - Revenue in USD
+     - Company, country
+
+   * - Tracking weather changes
+     - Precipitation level
+     - Location, sensor type
+
+   * - Recording fluctuations in housing prices
+     - Monthly rent price
+     - Location, currency
+
+.. _laravel-time-series-create:
+
+Create a Time Series Collection
+-------------------------------
+
+.. important:: Server Version for Time Series Collections
+
+   To create and interact with time series collections, you must be
+   connected to a deployment running MongoDB Server 5.0 or later.
+
+You can create a time series collection to store time series data.  
+To create a time series collection, create a migration class and
+add an ``up()`` function to specify the collection configuration.
+In the ``up()`` function, pass the new collection's name
+and the ``timeseries`` option to the ``Schema::create()`` method.
+
+.. tip::
+
+   To learn more about creating a migration class, see :ref:`laravel-eloquent-migrations`
+   in the Schema Builder guide.
+
+When setting the ``timeseries`` option, include the following fields:
+
+- ``timeField``: Specifies the field that stores a timestamp in each time series document.
+- ``metaField``: Specifies the field that stores metadata in each time series document.
+- ``granularity``: Specifies the approximate time between consecutive timestamps. The possible
+  values are ``'seconds'``, ``'minutes'``, and ``'hours'``.
+
+.. _laravel-time-series-create-example:
+
+Example
+~~~~~~~
+
+This example migration class creates the ``precipitation`` time series collection
+with the following configuration:
+
+- ``timeField`` is set to ``'timestamp'``
+- ``metaField`` is set to ``'location'``
+- ``granularity`` is set to ``'minutes'``
+
+.. literalinclude:: /includes/database-collection/time-series-migration.php
+   :language: php
+   :dedent:
+
+To verify that you successfully created the time series collection, call
+the ``Schema::hasCollection()`` method and pass the collection name as
+a parameter:
+
+.. code-block:: php
+
+   $result = Schema::hasCollection('precipitation');
+   echo $result;
+
+The ``hasCollection()`` method returns a value of ``1`` if the
+collection exists.
```

Comment:
`hasCollection()` actually returns a boolean, that is converted to `1` for `true`, or an empty string for `false`.

```suggestion
The ``hasCollection()`` method returns ``true`` if the collection exists.
```